### PR TITLE
TSDB: fix wrong initial value of tsidOrd in TimeSeriesIndexSearcher

### PR DESCRIPTION
**Pull request by weizijun**
This PR fixed the issue: https://github.com/elastic/elasticsearch/issues/85711

### CODE REVIEW

**Review by imotov — COMMENTED** [review]
Looks reasonable to me. @romseygeek, could you also take a look? 
@weizijun could you resolve merge conflicts meanwhile?

**Comment by weizijun** [comment]
> @weizijun could you resolve merge conflicts meanwhile?

done

**Comment by romseygeek** [comment]
@elasticmachine ok to test

**Review by csoulios — APPROVED** [review]
LGTM! I have a left a minor comment

File: server/src/test/java/org/elasticsearch/search/aggregations/timeseries/TimeSeriesIndexSearcherTests.java [hunk 1]
```diff
@@ -47,25 +48,18 @@ public class TimeSeriesIndexSearcherTests extends ESTestCase {
...
-                new SortField(DataStream.TimestampField.FIXED_TIMESTAMP_FIELD, SortField.Type.LONG)
-            )
-        );
-        RandomIndexWriter iw = new RandomIndexWriter(random(), dir, iwc);
+        RandomIndexWriter iw = getIndexWriter(dir);
 
         AtomicInteger clock = new AtomicInteger(0);
 
         final int THREADS = 5;
+        final int docCounts = 500;
```
[the remark this answers is not in the document]
Comment (weizijun) [reply server/src/test/java/org/elasticsearch/search/aggregations/timeseries/TimeSeriesIndexSearcherTests.java]: done

**Comment by romseygeek** [comment]
Thanks @weizijun!

**Pull request merged**